frontend/public/module/k8s/autocomplete.ts

```diff
@@ -9,6 +9,86 @@ import { K8sKind, K8sResourceKind } from '../../module/k8s';
 import { k8sListPartialMetadata } from '../../module/k8s/resource';
 import { getActiveNamespace } from '../../ui/ui-actions';
 
+export type AceSnippet = {
+  content: string;
+  name?: string;
+  guard?: string;
+  trigger?: string;
+  endTrigger?: string;
+  endGuard?: string;
+  tabTrigger?: string;
+};
+
+export const snippets = ImmutableMap<string, string>()
+  .set('metadata', `
+    metadata:
+      name: \${1}
+      namespace: \${2}
+      labels: 
+        \${3}
+      annotations:
+        \${4}
+  `)
```

Comment:
So this extra new-line character will be also present in the ace editor:
![alec](https://user-images.githubusercontent.com/1668218/42332332-fcc92b36-8077-11e8-8658-41ad33a00541.png)

Not sure if we want those extra new-lines. This applies to all the added snippets.

Reply:
Fixed it.

Reply:
But the extra line is still there. Shouldn't it be:
```js
  .set('metadata', `
    metadata:
      name: \${1}
      namespace: \${2}
      labels: 
        \${3}
      annotations:
        \${4}`
  )
```

Reply:
@jhadvig I prefer the backticks be on separate lines to make the template as clean as possible with correct indentation. [This line](https://github.com/openshift/console/pull/202/files#diff-1ff4faa1babdf14e37a86d526de07808R90) performs indentation/newline fixes.

Reply:
oh, I see :+1: